package-lock.json

```diff
@@ -1,2632 +0,0 @@
-{
-  "name": "openlmis-requisition",
```

Comment:
please do not remove this file, it is automatically generated and probably should be added to git ignore, however in this stage please leave this as it is

Reply:
This have been resolved @kmadej